src/proto/streams/send.rs

```diff
@@ -21,6 +26,14 @@ pub(super) struct Send {
     prioritize: Prioritize,
 }
 
+/// A value to detect which public API has called `poll_reset`.
+#[derive(Debug)]
+pub(crate) enum PollReset {
+    AwaitingHeaders,
+    #[allow(unused)]
```

Comment:
Why is this annotation needed? It seems to be used below.

Reply:
Ah, woops, I implemented just for `SendResponse` first, and so that variant was unused.